ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireAssertion.m

```diff
@@ -145,6 +145,7 @@ - (void)attemptToUseCacheItem:(ADTokenCacheItem*)item
     }
     
     //Now attempt to use the refresh token of the passed cache item:
+    BOOL isMultiresourceRefreshToken = item.multiResourceRefreshToken;
```

Comment:
There's something about this assumption that I find suspicious, but it's not your doing. Maybe in the future we'll add a test where we get back an RT after redeeming an MRRT, but I don't think that's a thing the server ever does to us.